localstack/ext/java/src/main/java/cloud/localstack/LambdaExecutor.java

```diff
@@ -91,8 +92,12 @@ public static void main(String[] args) throws Exception {
 
 				inputObject = DDBEventParser.parse(records);
 
+			} else if (records.stream().anyMatch(record -> record.containsKey("s3"))) {
```

Comment:
should we be checking if count >0 here? is it even possible?

Reply:
looks like no from what I can find; it just general interface for event generation sources which can have multiple records, but not in the case of s3